src/wdl/workflows/ValidateFileStatus/ValidateFileStatus.wdl

```diff
@@ -10,32 +10,38 @@ workflow ValidateGDCFileStatus {
     String workspace_name
     String workspace_project
     String sample_id
+    String sample_alias
+    String agg_project
+    String data_type
     Boolean delete = false
     File aggregation_path
   }
 
   String token_value = (read_lines(gdc_token))[0]
 
-  call validateFileStatus as file_status {
+  call tasks.ValidateFileStatus as file_status {
     input:
       program = program,
       project = project,
-      sample_id = sample_id,
+      sample_alias = sample_alias,
+      agg_project = agg_project,
+      data_type = data_type,
       gdc_token = token_value,
-      delete = delete
+      previous_task = true
   }
 
   call tasks.CreateValidationStatusTable as tsv {
-      input:
-        sample_id = sample_id,
-        file_state = file_status.file_state
+    input:
+      sample_id = sample_id,
+      file_state = file_status.file_state,
+      state = file_status.state
   }
 
   call tasks.UpsertMetadataToDataModel {
```

Comment:
From what I can tell, this task receives a tsv that has both `state` and `file_state`. But only the `file_state` column is getting populated in the workspace metadata. Should both columns be getting populated? And if so, do users know what the difference in these two columns means?